tic-tac-toe.js

```diff
@@ -1,7 +1,76 @@
 function TicTacToe() {
-
+  this.winConditions = [[0,1,2], [3,4,5], [6,7,8], [0,3,6], [1,4,7], [2,5,8],[0,4,8], [6,4,2]];
+  this.clickSpace();
 }
 
 TicTacToe.prototype = {
-  
-}
+  // check which player and return the appropriate mark
+  drawMark: function(player){
+    return player === 1 ? '🍍' : '🌺';
+  },
+
+  resetBoard: function(){
+    $(".square").each(function(){
+      this.innerText = "";
+    });
+  },
+
+  //check every square - if they are all filled, then do alert/game over TicTacToe
+  checkForTie: function() {
```

Comment:
I'm curious if this function is still getting called after a game has been won or lost and a board has been reset. It seems like there should be some additional logic which would prevent this method from being called in the case of a win or loss.